test/Tester/Placement/CustomPlacementTests.cs

```diff
@@ -40,6 +41,7 @@ public CustomPlacementTests(Fixture fixture)
 
             // sort silo IDs into an array
             this.silos = fixture.HostedCluster.GetActiveSilos().Select(h => h.SiloAddress.ToString()).OrderBy(s => s).ToArray();
+            this.siloAddresses = fixture.HostedCluster.GetActiveSilos().Select(h => h.SiloAddress).OrderBy(s => s.ToString()).ToArray();
```

Comment:
NIT: You don't need to call `.ToString()` since `SiloAddress` implements `IComparable`